ts/src/conductor.ts

```diff
@@ -193,6 +152,39 @@ export class Conductor {
     return createConductorPromise;
   }
 
+  setGossipParameters() {
+    const conductorConfig = readFileSync(
+      `${this.conductorDir}/conductor-config.yaml`,
+      "utf-8"
+    );
+    console.log("conductor config", conductorConfig);
+    const conductorConfigYaml = yaml.load(conductorConfig);
+    console.log("dump", conductorConfigYaml);
+    assert(conductorConfigYaml && typeof conductorConfigYaml === "object");
+    assert(
+      "network" in conductorConfigYaml &&
+        conductorConfigYaml.network &&
+        typeof conductorConfigYaml.network === "object"
+    );
+    assert("advanced" in conductorConfigYaml.network);
+    assert(conductorConfigYaml.network.advanced === null);
+    conductorConfigYaml.network.advanced = yaml.dump({
+      k2Gossip: {
+        initiateIntervalMs: 1000,
```

Comment:
I think this is probably the right thing to do. It will create unrealistic expectations when people go to run Holochain though, as we're seeing internally already.

Reply:
This should be less bad with the faster initiation thing, closing my comment